src/pk-main.c

```diff
@@ -173,6 +173,10 @@ main (int argc, char *argv[])
 	/* get values from the config file */
 	conf = g_key_file_new ();
 	conf_filename = pk_util_get_config_filename ();
+	if (conf_filename == NULL) {
+		g_print ("Cannot found config file.");
```

Comment:
Ok, let me change the message. ;-)
I didn't change to `g_printerr()` now to follow the current pattern inside pk-main.c.
I totally agree, but maybe it can be introduced by another patch. There are many occurrences of others `g_print()` for errors in other files either.